src/SourceBuild/patches/runtime/0001-Workaround-for-CreateRpmPackage-issue.patch

```diff
@@ -0,0 +1,23 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Alexander=20K=C3=B6plinger?= <alex.koeplinger@outlook.com>
+Date: Mon, 11 Nov 2024 14:24:15 +0100
+Subject: [PATCH] Workaround for CreateRpmPackage issue
+
```

Comment:
Please as a `Backport:` comment to an issue/PR per the [patch guidelines](https://github.com/dotnet/source-build/blob/main/Documentation/patching-guidelines.md#patch-guidelines).